src/app/shared/components/template/components/button-apple-sign-in/button-apple-sign-in.component.scss

```diff
@@ -0,0 +1,33 @@
+// Button is styled in line with Apple's guidleines:
+// https://developer.apple.com/documentation/sign_in_with_apple/displaying-sign-in-with-apple-buttons-on-the-web
+
+.apple-signin-btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: black;
+  color: white;
+  padding: 10px 30px 14px;
+  border-radius: 20px;
+  font-size: 16px;
+  font-weight: 600;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
+  border: none;
+  cursor: pointer;
+  transition: background-color 0.2s ease-in-out;
+  height: 40px;
+  min-width: 140pt;
```

Comment:
nit(non-blocking): the use of `pt` sizing seems a little strange as it's not used anywhere else. I can see on the link you shared for `40px` height the min-width is set at `210px` which might be a better value to use (and would correspond to the standard 1.33 px per pt which aligns with standard dpi values)

Reply:
Good point (/`pt`...).

Apple's guidelines are spread across a few different pages, but I've opted to set the minimum width to `140px`, which seems to be the required minimum width according to [this section](https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple#Button-size-and-corner-radius), and I think looks appropriate for when the button text string is shorter

<img width="377" alt="Screenshot 2025-03-27 at 16 46 29" src="https://github.com/user-attachments/assets/d91663c8-0840-4828-9fa3-bbd996e3b501" />